minio/console
# Inspect API

Inspect API  ( `mc admin inspect [--encrypt] <alias> <path>` )


Steps to Test:
- With Encryption  flag
   - `.enc` file will be generated along with a decryption key 
   - use `inspect` tool and enter the decryption key to get the `.zip` file
   - use `xl-meta` tool  to verify the contents of `.zip` file
- Without Encryption flag
   - use `xl-meta` tool  to verify the contents of `.zip` file.
  

API URL : `/api/v1/admin/inspect?file=test.txt%2Fxl.meta&volume=test-bucket&encrypt=true` 

Fixes https://github.com/miniohq/engineering/issues/479
